<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnprListManagementPage</class>
 <widget class="QWidget" name="AnprListManagementPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>862</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_licensePlate">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>License Plate</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="MyLineEdit" name="lineEdit_licensePlate">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>2</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_licenseType">
       <property name="text">
        <string>Plate Type</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="ComboBox" name="comboBox_licenseType"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="MyPushButton" name="pushButton_search">
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="TableView" name="tableView_license"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <widget class="QLabel" name="label_count">
       <property name="text">
        <string>Total 0 Items</string>
       </property>
       <property name="style" stdset="0">
        <string>transparent</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_page">
       <property name="text">
        <string>Total: Page:</string>
       </property>
       <property name="style" stdset="0">
        <string>transparent</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_firstPage">
       <property name="styleSheet">
        <string notr="true">QToolButton
{
	border: 0px;
	border-image: url(:/playback/playback/startpage_g.png);
}
QToolButton:hover
{
	border-image: url(:/playback/playback/startpage_b.png);
}
QToolButton:pressed
{
	border-image: url(:/playback/playback/startpage_g.png);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_previousPage">
       <property name="styleSheet">
        <string notr="true">QToolButton
{
	border: 0px;
	border-image: url(:/playback/playback/prevpage_g.png);
}
QToolButton:hover
{
	border-image: url(:/playback/playback/prevpage_b.png);
}
QToolButton:pressed
{
	border-image: url(:/playback/playback/prevpage_g.png);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="MyLineEditPage" name="lineEdit_page">
       <property name="maximumSize">
        <size>
         <width>100</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_nextPage">
       <property name="styleSheet">
        <string notr="true">QToolButton
{
	border: 0px;
	border-image: url(:/playback/playback/nextpage_g.png);
}
QToolButton:hover
{
	border-image: url(:/playback/playback/nextpage_b.png);
}
QToolButton:pressed
{
	border-image: url(:/playback/playback/nextpage_g.png);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton_lastPage">
       <property name="styleSheet">
        <string notr="true">QToolButton
{
	border: 0px;
	border-image: url(:/playback/playback/endpage_g.png);
}
QToolButton:hover
{
	border-image: url(:/playback/playback/endpage_b.png);
}
QToolButton:pressed
{
	border-image: url(:/playback/playback/endpage_g.png);
}</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="PushButtonPage" name="pushButton_go">
       <property name="text">
        <string>Go</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="MyPushButton" name="pushButton_template">
       <property name="text">
        <string>Download Template</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="MyPushButton" name="pushButton_add">
       <property name="text">
        <string>Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="MyPushButton" name="pushButton_deleteList">
       <property name="text">
        <string>Delete List</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="MyPushButton" name="pushButton_import">
       <property name="text">
        <string>Import</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="MyPushButton" name="pushButton_export">
       <property name="text">
        <string>Export</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="MyPushButton" name="pushButton_back">
       <property name="text">
        <string>Back</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyLineEdit</class>
   <extends>QLineEdit</extends>
   <header>mylineedit.h</header>
  </customwidget>
  <customwidget>
   <class>ComboBox</class>
   <extends>QComboBox</extends>
   <header>combobox.h</header>
  </customwidget>
  <customwidget>
   <class>MyPushButton</class>
   <extends>QPushButton</extends>
   <header>mypushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>MyLineEditPage</class>
   <extends>QLineEdit</extends>
   <header>mylineeditpage.h</header>
  </customwidget>
  <customwidget>
   <class>TableView</class>
   <extends>QTableView</extends>
   <header>tableview.h</header>
  </customwidget>
  <customwidget>
   <class>PushButtonPage</class>
   <extends>QPushButton</extends>
   <header>PushButtonPage.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
